.travis.yml

```diff
@@ -7,11 +7,15 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bower's cache
 
 env:
+  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
   - EMBER_TRY_SCENARIO=default
```

Comment:
so if I bump to the latest version then we won't need a default build, because that'll be covered in the `ember-release` scenario. Is that correct? If so that'll put us to 6 scenarios we're testing.